cartography/intel/aws/eks.py

```diff
@@ -0,0 +1,85 @@
+import logging
+
+from cartography.util import run_cleanup_job
+
+logger = logging.getLogger(__name__)
+
+
+def get_eks_clusters(boto3_session, region):
+    client = boto3_session.client('eks', region_name=region)
+    clusters = []
+    paginator = client.get_paginator('list_clusters')
+    try:
+        for page in paginator.paginate():
+            clusters.extend(page['clusters'])
+    except client.exceptions.ClientError as e:
+        # The account is not authorized to use this service in this region
+        # so we can continue without raising an exception
+        if e.response['Error']['Code'] == 'AccessDeniedException' \
+                and 'not authorized to use this service' in e.response['Error']['Message']:
+            logger.warn("{} in this region. Skipping...".format(e.response['Error']['Message']))
+        else:
+            raise(e)
```

Comment:
Can we change this to `raise`?  If I recall correctly we lose exception state if we do `raise(e)` but I could be wrong.

Reply:
Done!